src/include/clSPARSE.h

```diff
@@ -464,6 +464,15 @@ CLSPARSE_EXPORT clsparseStatus
 clsparseDdense2csr(const cldenseMatrix* A, clsparseCsrMatrix* csr,
                    const clsparseControl control);
 
+//SpM x SpM
+ CLSPARSE_EXPORT clsparseStatus
+        clsparseScsrSpGemm(
+        const clsparseCsrMatrix* sparseMatA,
+        const clsparseCsrMatrix* sparseMatB,
+              clsparseCsrMatrix* sparseMatC,
+        const clsparseControl control );
+
+
```

Comment:
I would prefer to wait for a separate pull request, Most of the work will be in test-blas3.cpp and benchmark functions and may be some effort needed in the SpMxSpM code.